pyproject.toml

```diff
@@ -5,3 +5,541 @@ requires = ["setuptools>=42", "wheel", "setuptools_scm"]
 show_error_codes = true
 pretty = true
 mypy_path = "typings_py"
+files = [
+    "sphinx_immaterial/**/*.py",
+    "tests/**/*.py",
+]
+exclude = "setup.py"
+
+[tool.black]
+include = '\.py$'
```

Comment:
We could set black to use a longer line-length because this is ignored in pylint.
```suggestion
[tool.black]
line-length = 100
include = '\.py$'
```
Running this change locally reformatted 43 files though.

Reply:
Yeah that is an annoying interaction between black and pylint, but anyway I moved the comment up to fix the issue.  I think it is better not to change the line length; choosing a different line length doesn't really solve the problem because then the lines will be longer to start with and an added pylint comment may still push it over the limit.

Reply:
That's fine. The whole idea for black is built on a "set & forget" approach.